examples/zone-printer/src/main.go

```diff
@@ -38,13 +56,74 @@ func main() {
 		log.Printf("info: determined zone: %q", zone)
 	}
 
-	log.Println("starting to listen on port 80")
-	http.HandleFunc("/", handle)
-	err := http.ListenAndServe(":80", nil)
-	log.Fatal(err)
+	finished := make(chan bool)
```

Comment:
this chan seems unused? promote the main handler to outside the goroutine perhaps?

Reply:
Done